pkg/controller/kubelet-config/kubelet_config_features.go

```diff
@@ -0,0 +1,207 @@
+package kubeletconfig
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	ignv2_2types "github.com/coreos/ignition/config/v2_2/types"
+	"github.com/golang/glog"
+	"github.com/imdario/mergo"
+	"github.com/vincent-petithory/dataurl"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/retry"
+	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+
+	osev1 "github.com/openshift/api/config/v1"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	ctrlcommon "github.com/openshift/machine-config-operator/pkg/controller/common"
+	mtmpl "github.com/openshift/machine-config-operator/pkg/controller/template"
+	"github.com/openshift/machine-config-operator/pkg/version"
+)
+
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
can you remove this? i think this might have been when this code was in `kubelet_config_controller.go`

Reply:
Removed.